app/javascript/apiCalls.js

```diff
@@ -0,0 +1,21 @@
+export function patchTimeSlot(slot, talk, csrfToken) {
+  const talkID = talk === null ? '' : talk.id.toString();
+
+  const data = JSON.stringify({
+    time_slot: {
+      program_session_id: talkID
+    }
+  });
+  
+  if (slot.update_path) {
+    return fetch(slot.update_path, {
+      method: "PATCH",
+      headers: {
+        "Content-Type": "application/json",
+        "X-CSRF-Token": csrfToken
+      },
+      credentials: "same-origin",
+      body: data
+    })
+  }
+}
```

Comment:
That red 'Ghostbuster' and return symbol there is no new line at end of file. It's good style/practice to have a new line at the end of files, and fortunately the text editor can handle that for you. I think you both use VS Code? First answer below should work, though you may have to open the file and save it. You can scroll through the files in this PR see which ones have no new line. 

https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files